src/features/search/components/LoadingSkeleton.tsx

```
@@ -8,65 +8,71 @@ interface LoadingSkeletonProps {
   lightMode = false,
 }) => {
-  const pulseVariants = {
-    initial: { opacity: 0.6 },
+  // Single animation definition for better performance
+  const shimmerVariants = {
+    initial: { backgroundPosition: '-500px 0' },
     animate: {
-      opacity: [0.6, 0.8, 0.6],
+      backgroundPosition: ['500px 0', '-500px 0'],
       transition: {
         repeat: Infinity,
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)